tests/core/pool.c

```diff
@@ -32,26 +25,24 @@ void test_core_pool__1(void)
 	int i;
 	git_pool p;
 
-	cl_git_pass(git_pool_init(&p, 1, 4000));
+	git_pool_init(&p, 1);
+	p.page_size = 4000;
 
 	for (i = 2010; i > 0; i--)
 		cl_assert(git_pool_malloc(&p, i) != NULL);
 
 	/* with fixed page size, allocation must end up with these values */
-	cl_assert_equal_i(1, git_pool__open_pages(&p));
-	cl_assert_equal_i(507, git_pool__full_pages(&p));
-
+	cl_assert_equal_i(590, git_pool__open_pages(&p));
```

Comment:
This looks like a non-trivial increase in pages. Is this down to the couple of pointer-sizes we take off of each page? If so, why isn't this adjusted for 32-bit systems like the other comparison?